app/code/Magento/ReviewGraphQl/Model/Resolver/Product/RatingSummary.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ReviewGraphQl\Model\Resolver\Product;
+
+use Exception;
+use Magento\Catalog\Model\Product;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Review\Model\Rating\GetProductRatingSummary;
+
+/**
+ * Product rating summary
```

Comment:
Better description is required. This is average rating for the product.

Reply:
Added